cmd/execute.go

```diff
@@ -213,8 +257,10 @@ func init() {
 	AddLoggingParameters(executeCmd)
 
 	// Prometheus
-	executeCmd.Flags().IntVar(&metricsPort, "metrics", 0, "Export via http prometheus metrics.")
-	executeCmd.Flags().StringVar(&metricsAddress, "metricsaddress", "", "If set other than empty string then Push otherwise pull, will be appended by port.")
+	executeCmd.Flags().StringVar(&metricsLevel, "metricslevel", "", "Export via http prometheus metrics. 0-off, 1-Pull, 2-Push without api, 3 push with api")
```

Comment:
update to reflect string names of enum, i.e. maybe something like
..."\n\t[0] nometrics (default)\n\t[1] pullmetric"
or similar so user can know something more understandable then the number can be written. 
it's ok write string like that, if you want to do it more advanced you can generate this help from the map like we do with loglevel

Reply:
Good suggestion, added